pallets/pool-system/src/lib.rs

```diff
@@ -656,6 +651,17 @@ pub mod pallet {
 			pool_id: T::PoolId,
 			epoch_id: T::EpochId,
 		},
+		PoolCreated {
```

Comment:
Can you add doc comments for both `PoolCreated` and `PoolUpdated` events?

Reply:
Forgot, thanks!

Reply:
Add admin and depositor to event.